qemu/tests/fio_perf.py

```diff
@@ -95,38 +121,53 @@ def run(test, params, env):
     :param env: Dictionary with test environment
     """
 
-    def fio_thread(session, run_cmd, cmd_timeout):
+    def fio_thread():
         """
         run fio command in guest
-
-        :param session: VM session
-        :param run_cmd: fio command
-        :param cmd_timeout: Timeout in seconds
         """
         session.cmd_status(run_cmd, cmd_timeout)
 
-    def _pin_vm_threads(vm, node):
+    def _pin_vm_threads(node):
```

Comment:
These several small function are defined inside function run, so they can only be used inside it, cann't be re-used or imported outside.